ansible/roles/load-balancer/meta/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+dependencies:
+    - { role: kubernetes }
```

Comment:
master role includes a dep for the kubernetes role, so line 3 should be unneeded.